addons/settings/stringtable.xml

```diff
@@ -53,7 +57,7 @@
             <Chinesesimp>模组设定选项...</Chinesesimp>
             <Portuguese>Opções ded Extensão...</Portuguese>
             <Russian>Настройки Дополнений...</Russian>
-            <French>Addon Options...</French>
+            <French>Ajuster les paramètres des addons.</French>
```

Comment:
This is shown in 3den. May want to check if `...` is not the style used by the base game strings there in French as well.